Client/Application/AppLaunchUtil.swift

```diff
@@ -174,4 +180,60 @@ class AppLaunchUtil {
                             category: .sync)
         }
     }
+
+    // MARK: - SKAdNetwork Postbacks
+
+   private struct ConversionValue {
+        //  fineValue - An unsigned 6-bit value ≥0 and ≤63. The app or the ad network defines the meaning of the fine conversion value.
+        //  coarseValue - An SKAdNetwork.CoarseConversionValue value of low, medium, or high. The app or the ad network defines the meaning of the coarse conversion value.
+
+        var fineValue: Int
+        var coarseValue: CoarseCoversionValue
+
+        enum CoarseCoversionValue {
+            case low
+            case medium
+            case high
+
+            @available(iOS 16.0, *)
+            var value: SKAdNetwork.CoarseConversionValue {
+                switch self {
+                case .low:
+                    return .low
+                case .medium:
+                    return .medium
+                case .high:
+                    return .high
+                }
+            }
+        }
+    }
+
+    private func adNetworkAttributionUpdateConversionInstallEvent(conversionValue: ConversionValue, completionHandler completion: ((Error?) -> Void)? = nil) {
```

Comment:
What is the purpose of the completion handler parameter?

Reply:
Hey, this parameter remained from the first version of the method and I forget to removed it. Sorry for that. I removed it now since it is not used :) Thank you!